tensorflow/lite/kernels/pooling3d.cc

```diff
@@ -442,6 +226,22 @@ TfLiteRegistration* Register_MAX_POOL_3D() {
   return &r;
 }
 
+}  // namespace builtin
+
+namespace custom {
+
+// The custom average pooling 3d op registered in custom_ops_register.h uses
```

Comment:
Shall we also keep Register_MAX_POOL_3D?

Reply:
Effectively, not sure why I didn't initially. Added the implementation for `custom::Register_MAX_POOL_3D`.